CONTRIBUTING.md

```diff
@@ -309,6 +310,8 @@ yarn dev --run-project /project/folder --record --key <key>
 
 #### Adding new Dependencies
 
+⚠️ There is a [bug in yarn](https://github.com/yarnpkg/yarn/issues/7734) that may cause issues adding a new dependency to a workspace. You can avoid this by downgrading yarn to 1.19.1 (temporarily downgrade using `npx yarn@1.19.1 workspace @packages/server add my-new-dep1`).
```

Comment:
Is it worth including the version range here? Otherwise I feel like we'll never delete this warning.

```suggestion
⚠️ There is a [bug in yarn up to v1.22.4](https://github.com/yarnpkg/yarn/issues/7734) that may cause issues adding a new dependency to a workspace. You can avoid this by downgrading yarn to 1.19.1 (temporarily downgrade using `npx yarn@1.19.1 workspace @packages/server add my-new-dep1`).
```

Reply:
I feel like this may imply that the issue is fixed after 1.22.4.